<?xml version="1.0"?>
<launch>
 
  <arg name="master_name" default="phantom"/>
  <arg name="robot_name" default="grips"/>
  <arg name="publish_rate" default="1000.0"/>	
  <arg name="ref_frame" default="world"/>
  <arg name="units" default="mm"/>
  <arg name="strategy" default="bubble_rate_pos"/> <!-- bubble_rate_pos / button_rate_pos / indexing -->
  <arg name="n_button" default="2"/>
  <arg name="buttons_topic" default="/teleop_controller/buttons"/>
   
   
  <node name="state_machine" pkg="teleop_strategies_controller" type="state_machine.py" output="screen">
	<param name="master_name" type="string" value="$(arg master_name)"/>
	<param name="robot_name" type="string" value="$(arg robot_name)"/>
    <param name="publish_rate" type="double" value="$(arg publish_rate)"/>
    <param name="ref_frame" type="string" value="$(arg ref_frame)"/>
    <param name="units" type="string" value="$(arg units)"/>
    <param name="strategy" type="string" value="$(arg strategy)"/>
    <param name="n_button" type="int" value="$(arg n_button)"/>
    <param name="buttons_topic" type="string" value="buttons_topic"/>
  </node>

     
</launch>
